snakecord/commands/client.py

```diff
@@ -1,8 +1,58 @@
+from functools import wraps
 from ..client import Client
 from .parsers import StringParser, FunctionArgParser
+from ..events import EventPusher
+from .commander import CommandInvokeEvent, CommandInvokeHandler
+from .command import Command
 
+from asyncio import get_event_loop
+
+class Commander(EventPusher):
+    def __init__(self, **kwargs):
+        loop = kwargs.get("loop") or get_event_loop()
```

Comment:
```suggestion
    def __init__(self, loop=None):
        loop = loop or get_event_loop()
```
What was I even thinking 🤔